src/Diagnostics.DataProviders/ConfigurationFactory.cs

```diff
@@ -90,6 +90,21 @@ protected override string GetValue(string prefix, string name)
                         return string.Empty;
                 }
             }
+            else if (prefix == "ContainerAppsMdm")
+            {
+                switch (name)
+                {
+                    case "MdmShoeboxEndpoint":
+                        return "https://az-int.metrics.nsatc.net";
```

Comment:
Looks like you're returning a hard-coded endpoint. Will this work across N-clouds?

Reply:
It's only for MockDataProviderConfigurationFactory. Actual endpoints are defined in a KV secret.
Oh, but I forgot settings for N-clouds. I will check.

Reply:
Oh nvm. I don't think it will make a difference then

Reply:
It turns out that we don't support container apps for national clouds yet. I will set it up once we get there.

Reply:
Can we use some garbled url instead of an actual one for mock purposes?
trying to avoid accidental leakage of these urls via code base